README-ZH_CN.md

```diff
@@ -1,6 +1,6 @@
 # TriliumNext Notes
 
-[English](https://github.com/TriliumNext/Notes/blob/master/README.md) | [Chinese](https://github.com/TriliumNext/Notes/blob/master/README-ZH_CN.md) | [Russian](https://github.com/TriliumNext/Notes/blob/master/README.ru.md) | [Japanese](https://github.com/TriliumNext/Notes/blob/master/README.ja.md) | [Italian](https://github.com/TriliumNext/Notes/blob/master/README.it.md)
+[🇬🇧 English](https://github.com/TriliumNext/Notes/blob/master/README.md) | [🇨🇳 Chinese](https://github.com/TriliumNext/Notes/blob/master/README-ZH_CN.md) | [🇷🇺 Russian](https://github.com/TriliumNext/Notes/blob/master/README.ru.md) | [🇯🇵 Japanese](https://github.com/TriliumNext/Notes/blob/master/README.ja.md) | [🇮🇹 Italian](https://github.com/TriliumNext/Notes/blob/master/README.it.md) | [🇪🇸 Spanish](https://github.com/TriliumNext/Notes/blob/master/README.es.md)
```

Comment:
Although I personally prefer adding flag icons to languages, it's generally not a good idea since flags represent countries, not languages. 

I had the same idea in the language selector but ended up scrapping it, so I think we should do the same in README for consistency. 

See https://www.flagsarenotlanguages.com/blog/why-flags-do-not-represent-language/ for more information.

Reply:
Change done.